modules/extensions/src/views/lite/components/debugger/index.tsx

```diff
@@ -74,14 +85,39 @@ export class Debugger extends React.Component<Props, State> {
   }
 
   loadEvent = async eventId => {
+    clearInterval(this.retryTimer)
+    const event = await this.fetchEvent(eventId)
+    if (!event) {
+      this.setState({ fetching: true })
+
+      this.retryTimer = window.setInterval(async () => {
+        await this.retryLoadEvent(eventId)
+      }, RETRY_PERIOD)
+    }
+    this.setState({ event, showEventNotFound: !event })
```

Comment:
showEventNotFound isn't really necessary, you could check if !this.state.event and check the state of fetching to deduce it